### DISCUSSION TITLE
libpag for visionOS

### DISCUSSION THREAD

**Discussion by Tongziqi** [Tencent/libpag#2246] [Q&A] [answered]
ld: building for 'visionOS', but linking in dylib (/Frameworks/libpag.framework/libpag) built for 'iOS'
clang: error: linker command failed with exit code 1 (use -v to see invocation)

请问下有没有支持visionOS的计划

**Comment by kevingpqi123** [comment, marked as answer]
由于人力有限，暂时没有支持 visionOS 的计划， pag 的源码是开源的，如果需要，欢迎参与共建
